apps/CommonApp/src/release_tests/rectButton/index.tsx

```diff
@@ -71,6 +71,7 @@ type ButtonProps = {
 function Button({ style, text }: ButtonProps) {
   const rectButtonStyles = [styles.button, style];
 
+  // eslint-disable-next-line no-alert
```

Comment:
What's wrong with `alert`?

Reply:
> Furthermore, alert is often used while debugging code, which should be removed before deployment to production.

This is a test app, it's never going to production. Actually, it's not going anywhere. #FreeAlert